### ISSUE TITLE
Readme states to use non module-path

### ISSUE THREAD

**Issue by glatzert** [PowerShell/Operation-Validation-Framework#19]
The Readme states about `Get-OperationValidation` to use `c:\temp\Modules\AddNumbers` this hides the fact, that the Path _must be_ part of a module search path. That should at least be mentioned. 
Or consider allowing a "LiteralPath" instead of the "ModulePath"

**Comment by devblackops** [comment]
You're right @glatzert. I plan to improve the readme (and documentation in general) to make it clearer what the project's goal is, how to use it, etc.
